### PR TITLE
document what country_osm_grid does

### PR DESCRIPTION
**Pull request by mtmail**
Document what `country_names` and `country_osm_grid` source files do. Also, since it's related, how `country_code` and `partition` values are set during place import.

Both files were created from an existing planet nominatim database and there likely were additional manual changes. Next time around we can add those manual steps.

### CODE REVIEW

File: data-sources/country-grid/README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+## Data files
+
+### `data/country_name.sql`
+
+Export from existing database table plus manual changes. `country_default_language_code` most likely taken from [https://wiki.openstreetmap.org/wiki/Nominatim/Country_Codes]()
+
+
+### `data/country_osm_grid.sql`
+
+See `sql/country_grid.sql`. It merges territories by country. Then uses `function.sql: quad_split_geometry` to split each country into multiple [Quadtree](https://en.wikipedia.org/wiki/Quadtree) polygons for faster point-in-polygon lookups.
```
Comment (lonvia) [inline data-sources/country-grid/README.md]: Any reason why you haven't moved the script (counrty_osm_grid.sql) into this directory this time?

File: data-sources/country-grid/README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+
+
+
+## Data files
+
+### `data/country_name.sql`
+
+Export from existing database table plus manual changes. `country_default_language_code` most likely taken from [https://wiki.openstreetmap.org/wiki/Nominatim/Country_Codes]()
+
```
[the remark this answers is not in the document]
Comment (mtmail) [reply data-sources/country-grid/README.md]: great, very helpful

File: data-sources/country-grid/README.md (hunk 1 above)
Comment (mtmail) [reply data-sources/country-grid/README.md]: good idea, moved now

**Comment by mtmail** [comment]
Reduced the image's dimensions. Didn't notice on the high DPI display.

**Comment by lonvia** [comment]
I had to fix up the linking in CMakeLists.txt. You were linking to docs/data-sources and then adding links inside, so that they ended up in the source directory. That left me and mkdocs confused.

**Pull request merged**